yt/visualization/base_plot_types.py

```diff
@@ -347,22 +347,15 @@ def _set_axes(self, norm: Normalize) -> None:
                 if MPL_VERSION < Version("3.5.0b"):
                     # no known working method to draw symlog minor ticks
                     # see https://github.com/yt-project/yt/issues/3535
+                    # and https://github.com/matplotlib/matplotlib/issues/21258
                     pass
                 else:
                     flinthresh = 10 ** np.floor(np.log10(norm.linthresh))
                     absmax = np.abs((norm.vmin, norm.vmax)).max()
                     if (absmax - flinthresh) / absmax < 0.1:
                         flinthresh /= 10
                     mticks = get_symlog_minorticks(flinthresh, norm.vmin, norm.vmax)
-                    if MPL_VERSION < Version("3.5.0b"):
-                        # https://github.com/matplotlib/matplotlib/issues/21258
-                        mticks = self.image.norm(mticks)
                     self.cax.yaxis.set_ticks(mticks, minor=True)
-
-            elif isinstance(norm, LogNorm):
-                self.cax.minorticks_on()
-                self.cax.xaxis.set_visible(False)
```

Comment:
this line used to be needed to workaround a bug in matplotlib, but it is fixed in all versions we currently support.